OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,77 +1,11 @@
-# Mount a host directory as a data volume
-
-Data volumes are designed to persist data, independent of the container’s lifecycle. The default location of the volume in container is under `/var/lib/docker/volumes`. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume. The volume will be used to store Database datafiles and WebLogic server domain files.This volume will be created on the host at `/scratch/DockerVolume/SOAVolume/`. Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
-
-To determine if a user already exists on your host system with uid:gid of 1000, run:
-
-    # getent passwd 1000
-
-If that returns a username (which is the first field), use that user for the `useradd` command below. If not, create the `oracle` user manually. 
-
-    # useradd -u 1000 -g 1000 oracle 
-
-Run the following commands as root:
-
-    # mkdir -p /scratch/DockerVolume/SOAVolume/
-    # chown 1000:1000 /scratch/DockerVolume/SOAVolume/
-    # chmod 700 /scratch/DockerVolume/SOAVolume/
-
-# Database
-
-You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
-
-Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleDatabase/README.md) under docker/OracleDatabase for details on how to build Oracle Database image.
-
-The DB image created through above step need to be retagged from `container-registry.oracle.com/oracle/database:12.2.0.1-ee`  to `oracle/database:12.2.0.1-ee` before continuing with next steps.
-
-$ docker tag container-registry.oracle.com/oracle/database:12.2.0.1-e  oracle/database:12.2.0.1-ee
-
-## Running Oracle Database in a Docker container
-
-Create an environment file db.env.list
-
-    ORACLE_SID=<DB SID>
-    ORACLE_PDB=<PDB ID>
-    ORACLE_PWD=<password>
-    
-Sample Data will look like this:
-
-    ORACLE_SID=soadb
-    ORACLE_PDB=soapdb
-    ORACLE_PWD=Welcome1
-
-Sample Command to Start the Database is as follows
-
-     $ docker run --name soadb  --network=SOANet -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
-     
-The above command starts a DB container attaching to a network and mounting a host directory as `/opt/oracle/oradata` for persistence. 
-It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
-
-
-## SOA Distributable version 12.2.1.3 Docker image Creation and Running
+## SOA Distributable version 12.2.1.x Docker image Creation and Running
```

Comment:
No image is permitted to be distributed by anyone besides Oracle  if it contains Oracle binaries. Not sure you're using the right word here.

Reply:
Corrected the wording.